### ISSUE TITLE
Behavior of default targets, i.e. prebuild --all

### ISSUE THREAD

**Issue by ralphtheninja** [prebuild/prebuild#205]
I'd like to get some input on what people think about this. So, what's the deal?

Currently when binaries are built using `prebuild --all`, it means the following targets will be built for `node` and `electron`:

From `node-abi`:

```js
var supportedTargets = [
  {runtime: 'node', target: '4.0.0', abi: '46', lts: false},
  {runtime: 'node', target: '5.0.0', abi: '47', lts: false},
  {runtime: 'node', target: '6.0.0', abi: '48', lts: new Date() < new Date(2018, 4, 18)},
  {runtime: 'node', target: '7.0.0', abi: '51', lts: false},
  {runtime: 'node', target: '8.0.0', abi: '57', lts: false},
  {runtime: 'node', target: '9.0.0', abi: '59', lts: new Date() < new Date(2019, 12, 31)},
  {runtime: 'electron', target: '0.36.0', abi: '47', lts: false},
  {runtime: 'electron', target: '1.1.0', abi: '48', lts: false},
  {runtime: 'electron', target: '1.3.0', abi: '49', lts: false},
  {runtime: 'electron', target: '1.4.0', abi: '50', lts: false},
  {runtime: 'electron', target: '1.5.0', abi: '51', lts: false},
  {runtime: 'electron', target: '1.6.0', abi: '53', lts: false},
  {runtime: 'electron', target: '1.7.0', abi: '54', lts: false},
  {runtime: 'electron', target: '1.8.0', abi: '57', lts: false}
]
```

This is fine and we use this for e.g. `leveldown`.

Now, we have gotten a PR that enables us to build for `NW.js` (previously known as `node-webkit`), see https://github.com/prebuild/prebuild/pull/203 for details.

A prerequisite for this to work is to make `node-abi` aware of `NW.js` and its corresponding ABI versions, see https://github.com/lgeiger/node-abi/pull/34 for details.

What is your opinion about making `prebuild --all` default to also include builds for `NW.js`? As I see it we have two options:

* The PR on `node-abi` includes `node-webkit` into the `supportedTargets` array -> `node-abi` has to release a new major in order to not mess up current behavior of `prebuild` -> If `prebuild` updates to new major of `node-abi` then `prebuild` should be released as a new major and once users update to this major they will all build for `node`, `electron` and `node-webkit`.
* The PR on `node-abi` includes `node-webkit` into an additional array (does not modify `supportedTargets`) -> `node-abi` can release a new minor and current behavior of `prebuild` is unchanged.

I personally would go for the second option. It would put the power into the users of `prebuild` to choose to build for `node-webkit` or not.

**Comment by daviwil** [comment]
I agree, I think it would be a potentially unwelcome surprise to have a bunch of new prebuild packages appear for node-webkit on a new release of a package that uses prebuild.  I'd go with option 2 so that those who really want it can opt in and then eventually add it to the default list if enough users request it.

**Comment by juliangruber** [comment]
Option 2 for me too, it would be great to make this configurable a la `prebuild --all=electron,node,nw`
